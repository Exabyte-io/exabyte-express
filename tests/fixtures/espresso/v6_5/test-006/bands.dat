version https://git-lfs.github.com/spec/v1
oid sha256:789f23cf21d3edbd7b4ec82289a618d02859c07230d6eb8dce1877470745d05c
size 11545
