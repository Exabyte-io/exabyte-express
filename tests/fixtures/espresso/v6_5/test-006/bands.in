version https://git-lfs.github.com/spec/v1
oid sha256:869db93df40a52c05e1684e7c3184c62650b4a50c1371e11d423ae2b54f37764
size 231
