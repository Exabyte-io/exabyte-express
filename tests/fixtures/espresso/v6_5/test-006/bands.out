version https://git-lfs.github.com/spec/v1
oid sha256:576eaca8163e5b3487be300077ff22ee1b7e1a52d5963a399f1d4e0feabde723
size 66414
